### ISSUE TITLE
A11y Swarm Byline Component

### ISSUE THREAD

**Issue by andrewscfc** [bbc/simorgh#5496]
**Is your feature request related to a problem? Please describe.**
Following completion of https://github.com/bbc/psammead/issues/3056 we need to complete an a11y swarm of the new `Byline` component to use in production for STYs and other pages in the future.
(see https://bbc.github.io/accessibility-news-and-you/accessibility-swarms)

**Describe the solution you'd like**
- Arrange and carry out dev swarm (2 person) as described above 
- Create and prioritise GitHub bug issue for any problems that are found, seeking guidance from a11y Champion as necessary

**Comment by simonsinclair** [comment]
Thanks for creating a ticket for this. We can link to it from the STY a11y swarm issue - the Byline and other STY-related components can be tested as part of the STY full a11y swarm.

**Comment by rhenshaw56** [comment]
**A11y Swarm**

https://bbc.github.io/accessibility-news-and-you/accessibility-news-and-testers
  - [X] Axe
  - [X] Design
  - [X] Colour Contrast
  - [X] Colour blindness
  - [X] Windows HCM
  - [X] FF Colour Preferences
  - [X] Text 200%, Page 200%
  - [X] Icons
  - [X] Heading Order
  - [X] Keyboard Navigation
      - Can't navigate to content with keyboard
  - [X] No JS (n/a)
     - no js not working on storybook

- [x] Screen readers
  - [x] ZoomText Magnifier/Reader
    - ~Content cannot be tabbed to~ - Expected behaviour
  - [x] Dragon NaturallySpeaking
  - [x] JAWS
  - [x] Read&Write 
  - [x] NVDA


_Mac, iOS, and Android:_ 
- [x] VoiceOver iOS (iPhone/iPad)
- [X] VoiceOver OS (Mac)
- [x] TalkBack (Chrome on Android)

Also, 
- [X] Check that the byline has list semantics
- [X] Check that the image is not acknowledged (this has no alt text so should not be)

**Comment by simonsinclair** [comment]
A dev a11y swarm on the Byline component has been completed by @HarveyPeachey and @rhenshaw56, and its alpha tag has been removed in https://github.com/bbc/psammead/pull/3117.
